helper/acctest/skip_on_error_test.go

```diff
@@ -0,0 +1,25 @@
+package acctest
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestSkipOnError(t *testing.T) {
```

Comment:
Nit: In Go testing, typically unit testing functions are named to match their code equivalent, e.g.

```suggestion
func TestSkipOnErrorContains(t *testing.T) {
```

Reply:
Of course!